pkg/bpm/bpm.go

```diff
@@ -237,6 +237,11 @@ func GetNsPath(pid uint32, typ NsType) string {
 
 // SetNS sets the namespace of the process
 func (b *ProcessBuilder) SetNS(pid uint32, typ NsType) *ProcessBuilder {
+	// NOTE: pid is 0 means don't need to enter namespace for this process.
+	if pid == 0 {
```

Comment:
I prefer judging outside this function and don't run `SetNS` at all...

Reply:
I agree with you, but need to add a lot of judgement, looks ugly 😨 
Do you have any idea about how to do judgment?

Reply:
I think we would never get pid of 0 from user's process. So here is must some logic that explicitly tell you "use pid 0 as this parameter".

Reply:
OK, I remove the `nsOptions` from ProcessBuilder and use it when build bec4459
PTAL again

Reply:
It's even worse than the original judgement (at least it doesn't break the `Builder` pattern)... 

You could use `SetNSOpt` directly to pass an option slice. If the slice is empty, then nothing would be done.